forge-gui/res/cardsfolder/upcoming/argivian_welcome.txt

```diff
@@ -0,0 +1,9 @@
+Name:Argivian Welcome
+ManaCost:W U
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Creature.powerGE4 | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature with power 4 or greater | SubAbility$ DBChoose | SpellDescription$ Destroy up to one target creature with power 4 or greater. Choose a nonland card in your hand. It perpetually gains flash. 
+SVar:DBChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Card.nonLand+YouOwn | ChoiceTitle$ Choose a nonland card in your hand | Amount$ 1 | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualFlash | Name$ Argivian Welcome's Perpetual Effect | Duration$ Permanent | SubAbility$ DBCleanup
+SVar:PerpetualLifelink:Mode$ Continuous | Affected$ Card.ChosenCard | AddKeyword$ Flash | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The chosen card perpetually gains flash.
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True
```

Comment:
You may not need Cleanup... try it without and see?

Reply:
OK, thanks for checking. I need to look into that code anyway

Reply:
Actually the card doesn't work at all. Does flash have to added as a static with CastWithFlash  ?

Reply:
nvm, the problem was that perpetuallifelink needed to be perpetualflash :z